CITATION.cff

```diff
@@ -0,0 +1,21 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "van Steen"
+  given-names: "Floyd"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
+- family-names: "Mirketou"
```

Comment:
My name is Marketou, can you change it please

Reply:
Solved dear anastasia